### ISSUE TITLE
Text Notifications not displaying whitespace in names

### ISSUE THREAD

**Issue by mueller-ma** [Freeyourgadget/Gadgetbridge#767]
#### Before opening an issue please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
- [x] If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)

#### Your issue is:
When someone calls me I see "FirstnameLastname" on my Mi Band 2.

#### Your wearable device is:
Mi Band 2

#### Your android version is:
Android 7.1.2
#### Your Gadgetbridge version is:
0.19.4

**Comment by danielegobbetti** [comment]
Is transliteration enabled? Does it happen also with the test notifications sent from debug menu of gadgetbridge?

**Comment by mueller-ma** [comment]
My issue is only in the contact's name, not in any message. I cannot reproduce it with the debug menu.

**Comment by ashimokawa** [comment]
@mueller-ma 
@tradej 
I think the white spaces are just very narrow, I tested "John Doe" in Debug and "JohnDoe", the difference not very striking.

**Comment by tradej** [comment]
@ashimokawa, you're right, there really is a space, but it's very narrow. The same thing is true for text messages.

Can this be changed somehow, for example through modifying the font file to make the space wider?

**Comment by ashimokawa** [comment]
No but we could duplicate spaces

**Comment by tradej** [comment]
I have tried "John  Doe" in the Debug mode, and it seems adequate. If they're easy enough to implement, double spaces are great.
